core/src/apps/ethereum/layout.py

```diff
@@ -139,6 +165,26 @@ async def confirm_typed_data_final() -> None:
     )
 
 
+def require_confirm_approve_tx(
+    address_bytes: bytes,
+    allowance: int,
+    network: EthereumNetworkInfo,
+    token: EthereumTokenInfo,
+) -> Awaitable[None]:
+    from trezor.ui.layouts import confirm_action
```

Comment:
Small nit (you could not know) - we are already using `confirm_action` in two functions in this file, and in this case we import it on top level instead of inside the functions. (Due to flash space savings)

So please move the import on top level.

(The same is actually true for `hexlify`, please do that as well.)

Reply:
Thanks for the feedback, fixes pushed.